askama_shared/src/input.rs

```diff
@@ -36,16 +36,15 @@ impl TemplateInput<'_> {
                 if template_args.is_some() {
                     return Err(CompileError::Static("duplicated 'template' attribute"));
                 }
-                let template = attr.parse_meta().map_err(|e| {
-                    CompileError::String(format!("unable to parse attribute: {}", e))
-                })?;
-                match template {
-                    syn::Meta::List(inner) => template_args = Some(inner),
-                    _ => {
-                        return Err(CompileError::Static(
-                            "attribute 'template' has incorrect type",
-                        ));
+
+                match attr.parse_meta() {
+                    Ok(syn::Meta::List(syn::MetaList { nested, .. })) => {
+                        template_args = Some(nested);
+                    }
+                    Ok(_) => {
+                        return Err("'template' attribute must be a list".into());
```

Comment:
```suggestion
                    Ok(_) => return Err("'template' attribute must be a list".into()),
```

To match the following one. Unless it was intentional? :)

Reply:
:+1: